shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -472,10 +472,15 @@ private void addPointerForIndex(MotionEvent event, int pointerIndex, int pointer
 
         packet.putLong(0); // obscured
 
-        InputDevice.MotionRange pressureRange = event.getDevice().getMotionRange(MotionEvent.AXIS_PRESSURE);
         packet.putDouble(event.getPressure(pointerIndex)); // pressure
-        packet.putDouble(pressureRange.getMin()); // pressure_min
-        packet.putDouble(pressureRange.getMax()); // pressure_max
+        if (event.getDevice() != null) {
+            InputDevice.MotionRange pressureRange = event.getDevice().getMotionRange(MotionEvent.AXIS_PRESSURE);
+            packet.putDouble(pressureRange.getMin()); // pressure_min
+            packet.putDouble(pressureRange.getMax()); // pressure_max
+        } else {
+            packet.putDouble(0.0); // pressure_min
```

Comment:
default should probably be 1.0

Reply:
A typical device uses pressure values ranging from 0 to 1

https://developer.android.com/reference/android/view/MotionEvent.html#getPressure(int)